src/python/install.sh

```diff
@@ -756,6 +756,36 @@ if [[ "${INSTALL_PYTHON_TOOLS}" = "true" ]] && [[ -n "${PYTHON_SRC}" ]]; then
     PY_MAJOR_MINOR_VER=${RUN_TIME_PY_VER_DETECT:7:4};
```

Comment:
can we bump patch version for this Feature?

Reply:
made changes to patch version.